rxn4chemistry/response_handler.py

```diff
@@ -67,13 +67,34 @@ def handle(self) -> Any:
             return {"response": self.response.text}
 
     def _print_error_logs(self) -> None:
+        """Note that logging has to be enabled, by default there is no output."""
         # error-specific logs
-        if self._payload is None or self._response_dict is None:
+        if self.response.status_code == 401 and len(self.response.history) > 0:
```

Comment:
It is a bit unclear to me what this is about and what it does (although I don't doubt that it is a useful change). 

Can you add as a comment what this is about? (I don't understand the comment at the end of the block).

Reply:
this is just an unfortunate diff, the line `if self._payload is None or self._response_dict is None:` is still there further down.

Reply:
I saw that, my comment is mainly about the wholly new `if` block 🙂

Reply:
Ah sorry now that I understand I expanded the comment.

For your awareness, there was a short time where this redirect happened for the default base_url, but it has been fixed now.

Reply:
perfect!